### ISSUE TITLE
Ahem font not pixel snapping clean enough(?) (linux?)

### ISSUE THREAD

**Issue by Gankro** [servo/webrender#1761]
Short on details since I'm about to sleep and don't want to forget this.

There's a bunch of reftests in gecko that are using the Ahem font to, basically, draw rectangles with very specific sizes. Enabling wr text breaks these. On linux (the reftests linked below) there's a visible shift left. On macos there's a visible shift up.

See the first ~30 tests here https://hg.mozilla.org/mozilla-central/raw-file/tip/layout/tools/reftest/reftest-analyzer.xhtml#logurl=https://queue.taskcluster.net/v1/task/careVUQ4TmuXvqUn9AkwxA/runs/0/artifacts/public/logs/live_backing.log&only_show_unexpected=1

**Comment by Gankro** [comment]
Related, a ton of the reftests have the *slightest* subpixel bleed to the left. I'm almost certainly going to fuzz these, but it might be the same issue.

See for instance 

```
file:///builds/worker/workspace/build/tests/reftest/tests/layout/reftests/bidi/bidi-006.html == file:///builds/worker/workspace/build/tests/reftest/tests/layout/reftests/bidi/bidi-006-ref.html
```

(near the top)

in https://hg.mozilla.org/mozilla-central/raw-file/tip/layout/tools/reftest/reftest-analyzer.xhtml#logurl=https://queue.taskcluster.net/v1/task/CljKfogYS4uurHw5atOWkQ/runs/0/artifacts/public/logs/live_backing.log&only_show_unexpected=1

**Comment by glennw** [comment]
We should definitely import ahem into wrench and make use of it for our tests.

There's a few known issues with Ahem in terms of rounding (see https://wiki.csswg.org/test/format#acceptable-test-formats) - but I assume that the Gecko tests don't have these issues since they are passing otherwise.

I guess the first step could be to get a single html test with ahem, and we'll get that pixel-perfect on all platforms, and then we can start using it for more tests?

**Comment by glennw** [comment]
Fixed in #1793
